src/scripts/Controller.js

```diff
@@ -407,6 +407,7 @@ export default class SitemapController {
 	 */
 
 	initSitemapValidation() {
+		const SITEMAP_ID_REGEXP = /^[a-z][a-z0-9_\$\(\)\+\-]+$/
```

Comment:
Move it to class field or module const. The idea is to define this regex once and then reuse in all validators.

Reply:
can I declare a SITEMAP_ID_REGEXP as const field inside a class? As I know, a constant field cannot be declared inside a class, it can be made static. If I do this, the following warning will appear:
![image](https://github.com/ispras/web-scraper-chrome-extension/assets/140788783/150db2b1-f737-42fa-a88c-86cd35912d51)

Reply:
My bad, indeed one cannot declare a const class property. I was thinking of extracting the regex so that subsequent modifications, if any, could be applied to one place in the code. Module const is totally fine for that.